### PR TITLE
A pass at revising README.rst

### PR DESCRIPTION
**Pull request by agrover**
Took a pass at tightening up and updating the README. Factual, stylistic, and wordsmithing comments welcome...

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -20,6 +24,10 @@ To install, check out the source, and use the included setup script, as::
 
    > python setup.py install
 
+You will also need to clone and install the following related Stratis
```
Comment (mulkieran) [inline README.rst]: Can you omit 'clone and'?

File: README.rst [hunk 2]
```diff
@@ -31,58 +39,48 @@ or::
    > stratis --version
 
 To run without installing, check out the source, change to the top directory
-and enter::
+and set ``PYTHONPATH`` environment variable to include library
```
Comment (mulkieran) [inline README.rst]: "set" -> "set the"

File: README.rst [hunk 3]
```diff
@@ -31,58 +39,48 @@ or::
...
-   > tox
-
-will cause some tests to be run in the tox virtual environment. Integration
-tests, which requires the CLI to connect with the Stratis daemon, will not
-be run.
-
-Note that stratis-cli is a Python 3 program, so it will be necessary for you
-to use the Python 3 variants of pytest and tox that are available on your
-system.
+  > STRATISD=../stratisd/target/debug/stratisd make dbus-tests
```
Comment (mulkieran) [inline README.rst]: The CI runs the stuff that can be run under tox. It's the only stuff that it can run.

File: README.rst (hunk 1 above)
Comment (agrover) [inline README.rst]: "clone and install" -> "obtain"?

Comment (mulkieran) [inline README.rst]: It's just that they don't really have to clone...they could use pip, or download a zip file from the arhcives, or something.

File: README.rst [hunk 4]
```diff
@@ -30,59 +39,54 @@ or::
...
+   > export PYTHONPATH="src:../stratisd-client-dbus/src:../into-dbus-python/src:../dbus-signature-pyparsing/src"
+   > python3 bin/stratis --help
 
-making sure that your PYTHONPATH environment variable is set to the CLI src
-directory.
-
-If you are not running the Stratis dbus service most actions will be
-unavailable, but all help menus should work properly.
+Since ``stratis`` uses stratisd's API, most operations will fail unless
+you also running the `Stratis daemon <https://github.com/stratis-storage/stratisd>`_.
```
Comment (mulkieran) [inline README.rst]: "you" -> "you are"

File: README.rst [hunk 5]
```diff
@@ -30,59 +39,54 @@ or::
...
 
 General Principles
 ------------------
-The CLI command-line is intended to have a simple structure to avoid
-ambiguities and constraints in parsing. For that reason, the initial portion
-of the command-line is a list of menu-selectors, each of which selects an
-option from the menu belonging to the previous selector. Once the final
-sub-menu is selected the remaining cli consists of options and arguments, only.
+``stratis`` uses positional arguments and subcommands to group related
+operations, and tries to use optional arguments sparingly.
```
Comment (trgill) [inline README.rst]: "and tries to use" -> "and uses" ?

not important - just a suggestion

File: README.rst [hunk 6]
```diff
@@ -30,59 +39,54 @@ or::
...
-using the Python argparse package.
+* The *parser* package handles configuring the command line parser, which uses
+  the Python `argparse <https://docs.python.org/3/library/argparse.html>`_ package.
 
-* The actions package contains code that mediates between the parser and the
-dbus. It contains functions that are automatically invoked by the
-parser during execution. Each function takes a parser Namespace argument,
-interprets it, and makes the necessary dbus calls.
+* The *actions* package receives valid commands from the parser package
+  and executes them, invoking D-Bus API calls as needed.  The parser
```
Comment (trgill) [inline README.rst]: "D-Bus API calls as needed" -> "D-Bus API as needed"

**Comment by agrover** [comment]
Thanks for reviews!!

**Pull request merged**